integration_tests/tests/apply/submitted_applications/view_application_overview_as_referrer.cy.ts

```diff
@@ -41,13 +46,19 @@ context('View submitted application overview', () => {
               label: 'Awaiting decision',
               description: 'All information has been received and the application is awaiting assessment.',
             }),
-            statusUpdateFactory.build({
-              updatedAt: DateFormats.dateObjToIsoDateTime(faker.date.past()),
-              updatedBy: externalUserFactory.build({ name: 'Anne Assessor' }),
-              name: 'moreInfoRequested',
+          ],
+          timelineEvents: [
+            timelineEventsFactory.build({
+              occurredAt: DateFormats.dateObjToIsoDateTime(faker.date.future()),
+              createdByName: 'Anne Other Assessor',
+              label: 'Awaiting decision',
+              body: 'All information has been received and the application is awaiting assessment.',
+            }),
+            timelineEventsFactory.build({
+              occurredAt: DateFormats.dateObjToIsoDateTime(faker.date.past()),
+              createdByName: 'Anne Assessor',
               label: 'More information requested',
-              description:
-                'More information about the application has been requested from the POM (Prison Offender Manager).',
+              body: 'More information about the application has been requested from the POM (Prison Offender Manager).',
```

Comment:
would it be worth making a few types of timelineEvents, some with `body` and some without?